org.eclipse.jdt.core.tests.model/src/org/eclipse/jdt/core/tests/model/NullAnnotationModelTests.java

```diff
@@ -1018,6 +1033,10 @@ public void testBug543304() throws Exception {
 
 			getWorkspace().build(IncrementalProjectBuilder.FULL_BUILD, null);
 			IMarker[] markers = client.getProject().findMarkers(IJavaModelMarker.JAVA_MODEL_PROBLEM_MARKER, true, IResource.DEPTH_INFINITE);
+			markers = List.of(markers).stream()
+					.filter(m -> !m.getAttribute(IMarker.MESSAGE, "")
+							.matches("The nullness annotation .* is not applicable for the primitive type.*"))
+					.toArray(IMarker[]::new);
```

Comment:
purpose of this test is to demonstrate that at 1.7 these errors do **not** occur.

I don't see any use in this test at 1.8, please remove to avoid future confusion.